amd/src/mod_hsuforum_button_animate.js

```diff
@@ -0,0 +1,14 @@
+define(['jquery'], function ($) {
+    return {
+        init: function (unfollow, following) {
```

Comment:
Why change the inner text based on mouse enter/exit events? If the user is already following the topic, what will happen?

Reply:
requirement asks for change of text on hover 
When hovering on the following state, the text must change to "Unfollow topic"

Reply:
Is that going to be done in a follow-up task?

Reply:
it's being done here